### PR TITLE
Branch delete medicine

### LINKED ISSUE #16: Able to delete medicine record
A User is able to remove a particular medicine existing in the application when required.

### PR DESCRIPTION
**Pull request by shxr3f**
Implemented methods that allows user to delete medicines from the records. (Resolves #16)
Also tried out some junit testing on the Parser Class methods! Please take a look at my code!

### CODE REVIEW

**Review by tanweili — APPROVED** [review]
Okay to merge on my end

File: src/main/java/seedu/duke/assets/MedicineList.java [hunk 1]
```diff
@@ -5,9 +5,17 @@
 public class MedicineList {
     private ArrayList<Medicine> medicines = new ArrayList<>();
 
+    public int size() {
```
Comment (kktai1512) [inline src/main/java/seedu/duke/assets/MedicineList.java:8]: maybe getSize would be a better name considering its a method?

File: src/main/java/seedu/duke/helper/Command.java [hunk 2]
```diff
@@ -67,4 +67,21 @@ public void addMedicine(MedicineList medicineList, String parameters) {
         medicineList.add(parameterArray);
         System.out.println("Medicine has been added");
     }
+
+    public void deleteMedicine(MedicineList medicineList, String stringIndex) {
```
Comment (kktai1512) [inline src/main/java/seedu/duke/helper/Command.java:71]: is the actual deleting not included?

should be calling medicineList.delete bla bla right

**Review by kktai1512 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/helper/Command.java (hunk 2 above)
Comment (shxr3f) [reply src/main/java/seedu/duke/helper/Command.java:71]: I think you missed it, its present on line 81.

**Comment by Anvitha-r** [comment]
looks good can be merged and closed!

**Comment by shxr3f** [comment]
Thanks for the reviews guys, I'll be merging the PR then, since there aren't any issues!

**Pull request merged**